src/utils/xQuest.cpp

```diff
@@ -2298,13 +2298,21 @@ class TOPPxQuest :
     progresslogger.startProgress(0, 1, "Matching to theoretical spectra and scoring...");
     vector< vector< CrossLinkSpectrumMatch > > all_top_csms;
 
+    Size spectrum_counter = 0;
+
 // TODO check if parallel code works fine
 #ifdef _OPENMP
 #pragma omp parallel for schedule(guided)
 #endif
     for (Size pair_index = 0; pair_index < spectrum_pairs.size(); ++pair_index)
     {
 
+#ifdef _OPENMP
+#pragma omp atomic
+#endif
+      spectrum_counter++;
+      cout << "Processing spectrum pair" << spectrum_counter << " / " << spectrum_pairs.size() << endl;
```

Comment:
@enetz maybe there should be brackets around this as I am sure a parallel `cout` is not going to end well ...